tip-wlan/charts/opensync-gw-cloud/templates/deployment.yaml

```diff
@@ -206,6 +206,10 @@ spec:
               value: {{ .Values.ethernetType.wanType }}
             - name: DEFAULT_WAN_NAME
               value: {{ .Values.ethernetType.wanName }}
+            - name: MQTT_BROKER_EXTERNAL_PORT
```

Comment:
If we can group the external variables together. Like `OVSDB_MANAGER`, `MQTT_SERVER_EXTERNAL` etc